src/app/app.extensions/example-card.card.js

```diff
@@ -1,7 +1,21 @@
 // For external API calls
 const axios = require("axios");
 
-exports.main = async (context = {}, sendResponse) => {
+const appConfig = {
```

Comment:
This replaces example-card.json. It also replaces serverless.json b/c `secrets` is moved into this json. We no longer need a file that links the appFunction to the app config because they live in the same file.

Reply:
I like this first version. I'm sure you all will discuss more and iterate on this, but pumped to see where it's going.